components/patient-data/api/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,15 @@
+phenotips.patientData.buttons.cancel=Cancel
+phenotips.patientData.buttons.import=Import
+phenotips.patientData.buttons.importFileSelect=Select a local file to be imported
+
+phenotips.patientData.jsonImportDialog.longWaitWarning=Importing may take a while, please be patient.
+phenotips.patientData.jsonImportDialog.importSuccess=Success!
+phenotips.patientData.jsonImportDialog.notAuthenticated=Failed: User authentication failed.
+phenotips.patientData.jsonImportDialog.notAuthorized=Failed: User not authorized.
+phenotips.patientData.jsonImportDialog.patientCreateFailed=Failed: Could not import patient data.
+
+phenotips.patientData.jsonImportDialog.fileUploadNotSupported=Your browser does not support file upload.<br>Please copy-paste import data in the box above.
```

Comment:
No markup or formatting in translation strings please, only plain text. Markup can be added as parameters if it's really needed. In this particular case, I think there's no need for a line break.